.github/workflows/builder.yml

```
@@ -0,0 +1,313 @@
+        if: needs.init.outputs.publish == "true"
+        shell: bash
+        run: |
+          export TWINE_USERNAME="$(twineUser)"
+          export TWINE_PASSWORD="$(twinePassword)"
+
+          twine upload dist/* --skip-existing
+
+  build_base:
+    name: Build ${{ matrix.arch }} base core image
+    needs: init
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch: ${{ fromJson(needs.init.outputs.architectures) }}
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+
+      - name: Set up Python ${{ env.DEFAULT_PYTHON }}
+        if: needs.init.outputs.channel == "dev"
+        uses: actions/setup-python@v2.2.1
+        with:
+          python-version: ${{ env.DEFAULT_PYTHON }}
+
+      - name: Adjust nightly version
+        if: needs.init.outputs.channel == "dev"
+        shell: bash
+        run: |
+          python3 -m pip install packaging
+          python3 -m pip install .
+          python3 script/version_bump.py nightly
+          version="$(python setup.py -V)"
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ secrets.GIT_USER }}
+          password: ${{ secrets.GIT_TOKEN }}
+
+      - name: Build base image
+        uses: home-assistant/builder@2021.04.2
+        with:
+          args: |
+            $BUILD_ARGS \
+            --${{ matrix.arch }} \
+            --target /data \
+            --with-codenotary "${{ secrets.VCN_USER }}" "${{ secrets.VCN_PASSWORD }}" "${{ secrets.VCN_ORG }}" \
+            --validate-from "${{ secrets.VCN_ORG }}" \
+            --generic ${{ needs.init.outputs.version }}
+
+  build_machine:
+    name: Build ${{ matrix.arch }} machine core image
+    needs: ["init", "build_base"]
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        machine:
+          - generic-x86-64
+          - intel-nuc
+          - odroid-c4
+          - odroid-n2
+          - odroid-xu
+          - qemuarm
+          - qemuarm-64
+          - qemux86
+          - qemux86-64
+          - raspberrypi
+          - raspberrypi2
+          - raspberrypi3
+          - raspberrypi3-64
+          - raspberrypi4
+          - raspberrypi4-64
+          - tinker
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ secrets.GIT_USER }}
+          password: ${{ secrets.GIT_TOKEN }}
```

We don't need GitHub PAT's anymore.

```suggestion
          username: ${{ github.repository_owner }}
          password: ${{ secrets.GITHUB_TOKEN }}
```